client/login/magic-login/request-login-email-form.jsx

```diff
@@ -75,6 +75,9 @@ class RequestLoginEmailForm extends React.Component {
 						onDismissClick={ this.onNoticeDismiss }
 						status="is-error" />
 				}
+				<h1 className="magic-login__formheader">
```

Comment:
@stephanethomas That's what I originally had, but noticed there was already a `magic-login__formheader` in the stylesheet (being unused) that already had all the styles necessary. So I decided to use it rather than making changes to both the component and stylesheet.